lua/entities/gmod_door_interior/modules/cl_cordon.lua

```diff
@@ -57,16 +52,13 @@ function ENT:UpdateCordon()
 			end
 		end
 		if check then
-			--if not self.props[v] then
-				--print("enter",v)
-			--end
+			-- if not self.props[v] then
+			-- 	print("enter",v)
+			-- end
```

Comment:
Wondering if this should be removed or if it should stay. You always ask me to remove comments like these, after all.

Reply:
I like to keep this one here because it comes in handy whenever I'm debugging the cordon and I can never remember the right bit of code and where to put it

Reply:
Alright then, sounds good.